spec/helpers/application_helper/buttons/miq_ae_spec.rb

```diff
@@ -0,0 +1,68 @@
+describe ApplicationHelper::Button::MiqAe do
+  let(:session) { Hash.new }
+  let(:view_context) { setup_view_context_with_sandbox({}) }
+
+  before do
+    Tenant.seed
+    feature_list = %w(
```

Comment:
what about to create trait in FactoryGirl ?
than you can have 

`FactoryGirl.create(:user, :with_features)` and you can remove lost of dups here in PRs. 

https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#traits